_studio/mfx_lib/encode_hw/h264/include/mfx_h264_encode_hw_utils.h

```diff
@@ -1287,12 +1291,12 @@ namespace MfxHwH264Encode
         virtual mfxStatus Init(MfxVideoParam  & video) = 0;
         virtual mfxStatus Reset(MfxVideoParam  & video) = 0;
         virtual void Close() = 0;
-        virtual void PreEnc(const BRCFrameParams& par, std::vector<VmeData *> const & vmeData) = 0;
-        virtual mfxU8 GetQp(const BRCFrameParams& par) = 0;
-        virtual mfxU8 GetQpForRecode(const BRCFrameParams& par, mfxU8 curQP) = 0;
-        virtual mfxF32 GetFractionalQp(const BRCFrameParams& par) = 0;
-        virtual void SetQp(const BRCFrameParams& par, mfxU32 qp) = 0;
-        virtual mfxU32 Report(const BRCFrameParams& par, mfxU32 dataLength, mfxU32 userDataLength, mfxU32 maxFrameSize, mfxU32 qp) = 0;
+        virtual void PreEnc(BRCFrameParams& par, std::vector<VmeData *> const & vmeData) = 0;
```

Comment:
Please do not remove 'const'

Reply:
On which line does it change? Anyway, too many ```const``` qualifiers removed